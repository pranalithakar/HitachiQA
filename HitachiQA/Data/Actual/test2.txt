44
88
33
24
36