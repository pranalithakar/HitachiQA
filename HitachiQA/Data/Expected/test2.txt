Test
ALL
The
Things
At 
Once